kore/src/Kore/Step/Simplification/Equals.hs

```diff
@@ -441,10 +441,14 @@ termEqualsAnd p1 p2 =
                 Logic.scatter
             . sequence
         equalsPredicate =
```

Comment:
Maybe this should be renamed to `equalsPattern`?

Reply:
I took care of this as my clean-up commit. I couldn't find anything else to refactor.